app/javascript/schools/settings/components/SchoolCustomize__LinkComponent.res

```diff
@@ -0,0 +1,258 @@
+open SchoolCustomize__Types
+
+type action =
+  | SetEditing(bool)
+  | UpdateTitle(string)
+  | UpdateUrl(string)
+  | SetError(bool)
+  | SetUpdating(bool)
+
+type state = {
+  title: string,
+  url: string,
+  editing: bool,
+  error: bool,
+  updating: bool,
+}
+
+let str = React.string
+
+let inputClasses = "bg-white border border-gray-400 rounded py-2 px-3 focus:outline-none focus:ring-2 focus:ring-indigo-500 placeholder-gray-500"
+
+let deleteIconClasses = deleting => deleting ? "fas fa-spinner fa-pulse" : "far fa-trash-alt"
+
+module DestroySchoolLinkQuery = %graphql(`
+  mutation DestroySchoolLinkMutation($id: ID!) {
+    destroySchoolLink(id: $id) {
+      success
+    }
+  }
+  `)
+
+module SortSchoolLinkQuery = %graphql(`
+  mutation SortSchoolLinksMutation($id: ID!,$direction:MoveDirection!) {
+    sortSchoolLinks(id:$id,direction:$direction) {
+      links {
+        id
+        title
+        url
+        kind
+        sortIndex
+     }
+    }
+  }
+  `)
+
+let handleMoveLink = (~id, ~direction, ~dispatch, ~moveLinkCB) => {
+  dispatch(SetUpdating(true))
+  SortSchoolLinkQuery.make({id: id, direction: direction})
```

Comment:
Use the `fetch` function instead of `make` in queries, as `make` is deprecated after GraphQL PPX upgrade.